definitions/ext-host/definition.yml

```diff
@@ -50,7 +50,32 @@ synthesis:
       encodeIdentifierInGUID: true
       conditions:
       - attribute: eventType
-        value: AnsibleDiskIOSample 
+        value: AnsibleDiskIOSample
+
+    # logging data
+    - identifier: hostname
+      name: hostname
+      conditions:
+        - attribute: 'service_name'
+          present: false
+        - attribute: 'container.name'
+          present: false
+
+    - identifier: host
+      name: host
+      conditions:
+        - attribute: 'service_name'
+          present: false
+        - attribute: 'container.name'
+          present: false
+
+    - identifier: host.name
+      name: host.name
+      conditions:
+        - attribute: 'service_name'
+          present: false
+        - attribute: 'container.name'
+          present: false
```

Comment:
I see. The problem is that we don't have a way of prioritizing rules (yet). So, we couldn't ensure that `host` would be preferred. Isn't there something else you could use to match one exact rule?

Reply:
I checked with @naxhh and we could try to do something here.

The prioritization within the same definition can be achieved as they appear in the file. So, if you want `host` to be prioritized, that rule needs to be on the top.

Then, in order to avoid other clashes with other rules and definitions, you could try to narrow the scope of the rule. Is this coming from Log event? Then we can add that as a condition (eventType). Even also you can make use of this condition: `"newrelic.source": "api.logs"`. 

Finally, we offer a way to test your rules against test data. See more info [here](https://github.com/newrelic/entity-definitions#testing). Those payloads you paste would be nice examples to validate your rules match.